internal/pkg/cli/env_init.go

```diff
@@ -356,8 +439,58 @@ func BuildEnvInitCmd() *cobra.Command {
 			return opts.Execute()
 		}),
 	}
-	cmd.Flags().StringVarP(&vars.EnvName, nameFlag, nameFlagShort, "", envFlagDescription)
-	cmd.Flags().StringVar(&vars.EnvProfile, profileFlag, "", profileFlagDescription)
+	cmd.Flags().StringVarP(&vars.Name, nameFlag, nameFlagShort, "", envFlagDescription)
```

Comment:
I wonder if the flag changes to the command should actually happen at the end? once the feature is ready to be enabled.

Basically, we should strive to have the commits in our trunk branch always be deployable. So that in case we need to do an emergency fix, we can just make the change and do a release. Right now, the command would be broken.

Reply:
I'm down to keep these changes but maybe commenting them for now until `Ask` is implemented, what do you think?

Reply:
Agree. I'll comment this out